aprslib/parsing/__init__.py

```diff
@@ -159,7 +180,7 @@ def _try_toparse_body(packet_type, body, parsed):
     # ^ - unused
     # } - 3rd party traffic
     if packet_type in '#$%)*<?T[}':
-        raise UnknownFormat("format is not supported")
+        raise UnknownFormat('format is not supported: {0}'.format(unsupported_formats[packet_type]))
```

Comment:
There doesn't seem to  be a default value here. If `packet_type` is not in the map this will raise a `KeyError`